great_expectations/render/renderer/inline_renderer.py

```diff
@@ -153,9 +156,10 @@ def _get_renderer_atomic_rendered_content(
                     f"renderer_name: {renderer_name} was not found in the registry for expectation_type: {expectation_type}"  # noqa: E501 # FIXME CoP
                 )
 
-            assert isinstance(renderer_rendered_content, RenderedAtomicContent), (
+            assert isinstance(renderer_rendered_content, (RenderedAtomicContent, list)), (
```

Comment:
Might be good to reuse that TypeGuard for stuff like this, but I'm not sure why we don't trust our types to be right at runtime. So maybe that's overkill.